fc/protocol.py

```diff
@@ -27,6 +20,7 @@
 from .language.statements import Assign  # noqa: E402
 from .locatable import Locatable  # noqa: E402
 from .parsing import actions  # noqa: E402
+from .plotting import create_plot  # noqa: E402
```

Comment:
And now we've moved the matplotlib import, all these noqa markers should be redundant.